recipes/beyla-golden-signals/gce/beyla-config.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Beyla can be configured to listen to applications running on multiple ports.
+# Beyla can also be configured globally using environment variables.
+# Environment variables have priority over the properties in the config file.
+open_port: 8080,8082,8083
+trace_printer: text
+protocol: grpc
+
+otel_metrics_export:
+  endpoint: http://0.0.0.0:4317
+  # default includes all possible instrumentations "*"
+  instrumentations: ["http", "grpc"]
+  # default only includes "application"
+  features: ["application", "application_process"]
```

Comment:
Is application_process duplicative of what the ops-agent already provides?

Reply:
Looking at the list and the generated metrics, they are slightly different in terms of what they measure/report.

Major difference is that the metrics reported by Beyla can be on a per-process level. Ops Agent seems to report it on the VM level.
For instance I can get cpu_utilization on per process basis with beyla, but Ops agent seems to report only the overall utilization of the VM.

Reply:
Seems like it should have per-process metrics? https://cloud.google.com/monitoring/api/metrics_opsagent#agent-processes

Reply:
Ah, missed the `/processes/` prefix. Yes, in this case most metrics are overlapping.